src/daft-dsl/src/lit/mod.rs

```diff
@@ -876,18 +873,6 @@ mod test {
         roundtrip(1u64)?;
         roundtrip(1f64)?;
         roundtrip("test".to_string())?;
-        roundtrip(IOConfig::default())?;
-        roundtrip(CountMode::All)?;
-        roundtrip(ImageFormat::PNG)?;
-        roundtrip(ImageMode::L)?;
-        roundtrip(ImageMode::LA)?;
-        roundtrip(ImageMode::RGBA)?;
-        roundtrip(ImageMode::L16)?;
-        roundtrip(ImageMode::LA16)?;
-        roundtrip(ImageMode::RGB16)?;
-        roundtrip(ImageMode::RGBA16)?;
-        roundtrip(ImageMode::RGB32F)?;
-        roundtrip(ImageMode::RGBA32F)?;
```

Comment:
shouldnt these still all be round trippable?

Reply:
Since we are storing these as Python objects in the round trip, we can't test them on the Rust side anymore unfortunately.